packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -26,7 +26,9 @@ const Redirect = ({
     let redirected_to_route = false;
     const action_param = url_params.get('action');
     const code_param = url_params.get('code') || verification_code[action_param];
+    sessionStorage.verification_code = code_param;
     const ext_platform_url = url_params.get('ext_platform_url');
+    const is_next_wallet = localStorage.getObject('FeatureFlagsStore')?.data.next_wallet;
```

Comment:
```suggestion
    const { is_next_wallet_enabled } = useFeatureFlags();
```

Will it work?

Reply:
Tried this initially but doesn't seem to work. The issue seems to be that the redirection from the email url loads faster than the `is_next_wallet_enabled` boolean causing it to always be false. Which is why other redirections from the redirect.jsx file used methods such as WS.wait or calling values from the local storage instead to check the condition.

Which is why the workaround for it was instead, to use the `next_wallet` boolean value stored in local storage